README.md

```diff
@@ -12,16 +12,6 @@ A simple Elixir package for implementing [reCAPTCHA] in Elixir applications.
 
 [reCAPTCHA]: http://www.google.com/recaptcha
 
-## Migration from 1.x
-
-### Breaking Changes
-
-1. Template functionality is now in a separate module: `Recaptcha.Template`. Please note: in future templating may move to a Phoenix specific package.
-2. `verify` API has changed, see the code for documentation of the new API.
-
-Most other questions about 2.x should be answered by looking over the documentation and the code. Please raise an issue
-if you have any problems with migrating.
```

Comment:
Maybe we should keep this but change it to: "Migration from 1.x -> 2.x"

Reply:
@vorce Sounds good to me. Initially deleted it as we're completely moving in a different direction from the original library but I guess it makes sense to keep this section for users that do happen to be using those versions.

I have added this section back in [this commit](https://github.com/remoteoss/recaptcha/pull/1/commits/947456c36cd8abc12e632db4bb809a94fca8a10b) 🙂